Source/WebCore/inspector/agents/InspectorAnimationAgent.cpp

```diff
@@ -588,7 +588,7 @@ String InspectorAnimationAgent::findAnimationId(WebAnimation& animation)
     return nullString();
 }
 
-WebAnimation* InspectorAnimationAgent::assertAnimation(Inspector::Protocol::ErrorString& errorString, const String& animationId)
+RefPtr<WebAnimation> InspectorAnimationAgent::assertAnimation(Inspector::Protocol::ErrorString& errorString, const String& animationId)
```

Comment:
This seems wrong. We only want allocating methods or methods marked "protected" to return smart pointers, afaik. I also doubt this is needed now that you've captured the return value in a smart pointer at the call sites.